.github/workflows/ci_cd_release.yml

```
@@ -83,7 +101,7 @@ jobs:
       pull-requests: write
     runs-on: ubuntu-latest
     steps:
-      - uses: ansys/actions/doc-deploy-changelog@main
+      - uses: ansys/actions/doc-deploy-changelog@feat/check-actions-security-action
```

```suggestion
      - uses: ansys/actions/doc-deploy-changelog@main
```
Needs to be reverted before merging